knowledge-base/changes-on-items-in-report-events-are-not-applied.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Changes on items in report events are not applied
+description: Changes on items in report events are not applied.
+type: how-to
+page_title: Changes on items in report events are not applied
+slug: changes-on-items-in-report-events-are-not-applied
+res_type: kb
+---
+
+## Description
+
+Changes on items in report events are not applied.  
```

Comment:
Why does this line exist? It simply repeats the title, while it doesn't give any additional details. Remove it.